docs/install.md

```diff
@@ -58,8 +69,8 @@ helm upgrade kedahttp ./charts/keda-http-operator \
     --namespace ${NAMESPACE} \
     --create-namespace \
     --set image=localhost:32000/keda-http-operator \
-	--set images.scaler=localhost:32000/keda-http-scaler \
-	--set images.interceptor=localhost:32000/keda-http-interceptor
+  --set images.scaler=localhost:32000/keda-http-scaler \
```

Comment:
@khaosdoctor small nit: the new spacing doesn't look lined up still. Can you adjust?

Reply:
Sure! Will fix it right away